src/vm/ast/parser/mod.rs

```diff
@@ -68,7 +68,7 @@ pub fn lex(input: &str) -> ParseResult<Vec<(LexItem, u32, u32)>> {
     let lex_matchers: &[LexMatcher] = &[
         LexMatcher::new(r##""(?P<value>((\\")|([[:print:]&&[^"\n\r\t]]))*)""##, TokenType::StringLiteral),
```

Comment:
Should we also update this regex?
`\n\r\t` to `\n\t`

Reply:
Addressed in: cab43a12a5a4395272d37c43d68610f47939aa5f